app/models/concerns/blacklight/user.rb

```diff
@@ -28,4 +28,10 @@ def document_is_bookmarked?(document)
   def existing_bookmark_for(document)
     bookmarks_for_documents([document]).first
   end
+
+  ##
+  # @return [String] a user-displayable login/identifier for the user account
+  def to_s
+    email
```

Comment:
I think `email` is something we get from devise, and I think the goal of putting it in the generator was not to tie others to that particular implementation?

Reply:
Does inheriting it tie them to that implementation any more than generating it into their code?

The problem I'm encountering with this approach is that it's awkward to introduce an alternate implementation anywhere other than the application.

Reply:
see, e.g.: [`Blacklight::AccessControls::User#user_key`](https://github.com/projectblacklight/blacklight-access_controls/blob/master/lib/blacklight/access_controls/user.rb#L16-L21) and `Hyrax::User#display_name`. Arguably, both of these should provide an alternate default `#to_s` implementation, but they can't achieve this without editing files in the application's tree.